### DISCUSSION TITLE
Turn of google recaptcha

### DISCUSSION THREAD

**Discussion by nikolatepavacy** [uvdesk/community-skeleton#734] [Q&A]
Hello community,

I turned od google recaptcha accidentally and now I can't sign in with any account.
Is it possible to turn of google recaptcha in config file?
I tried inserting is_google_captcha_enabled = false in config/packages/uvdesk.yaml, but then I get Symphony Error.

**Comment by manavgangwani** [comment]
Hello, my name is Manav Gangwani. If you are an end user trying to bypass or turn off reCAPTCHA on a website, it's important to note that this is generally not possible and unethical. reCAPTCHA is used to prevent spam and abuse, and bypassing it violates terms of service and potentially legal standards.

**In Google reCAPTCHA Admin Console:**

- Go to the Google reCAPTCHA Admin Console: Google reCAPTCHA Admin
- Log in with the account you used to set up reCAPTCHA.
- Select the site key you want to disable.
- Delete the site key if you want to completely remove reCAPTCHA from that site, or adjust the settings as needed.

**Log in to your WordPress admin panel**

- Navigate to the reCAPTCHA plugin you are using (e.g., reCAPTCHA by BestWebSoft, Contact Form 7 if it has reCAPTCHA enabled).
- Disable or remove the plugin if you no longer want to use reCAPTCHA. Alternatively, if you only want to disable it temporarily, you can just deactivate the plugin.
- Save changes and check your website to ensure reCAPTCHA is no longer active.

**For Developers During Development/Testing**

If you are a developer and want to disable reCAPTCHA during development or testing, you can:
- Use reCAPTCHA test keys provided by Google. These keys allow you to test without triggering reCAPTCHA challenges.
- Site key: 6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
- Secret key: 6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
- Comment out or conditionally disable the reCAPTCHA code in your development environment.
